tests/ceph_volume/test_ceph_volume_lvm_cli_validations.py

```diff
@@ -0,0 +1,49 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import ConfigError
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Passing incorrect arguements to ceph-volume lvm
+    prepare, activate and create command should not
+    cause crashes. User understandable error/warning shuld be
+    displayed
+    """
+    # Get configs
+    configs = kw.get("config")
+
+    # Get specs from config
+    data = configs.get("device_path")
+    if not data:
+        raise ConfigError("Mandatory config 'data' is not provided")
+    osd_id = configs.get("osd_id")
+    if not osd_id:
+        raise ConfigError("Mandatory config 'osd_id' is not provided")
+    osd_fsid = configs.get("osd_fsid")
+    if not osd_fsid:
+        raise ConfigError("Mandatory config 'osd_fsid' is not provided")
+
+    # Get the installer node
+    node = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Set CephAdm object
+    cephadm = CephAdm(node).ceph_volume.lvm
```

Comment:
this is not `cephadm` object, it's `lvm` object

Reply:
Done